apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -117,7 +117,10 @@ internal TargetedSessionPool(SessionPool parent, DatabaseName databaseName, bool
                         Labels = { parent.Options.SessionLabels }
                     }
                 };
-
+                if (databaseRole != null)
```

Comment:
I'd just use `CreatorRole = databaseRole ?? ""` in the object initializer above.